### DISCUSSION TITLE
Airconnect of synology will stop when TV with supporting airplay protocol start.

### DISCUSSION THREAD

**Discussion by githubgj** [eizedev/AirConnect-Synology#56] [Q&A]
Airconnect of synology will stop when TV with supporting airplay protocol start.If I want to restart the airconnect app,I must ture off TV,please help me,thank you

**Comment by eizedev** [comment]
Do you play your music on the TV and then when you turn on the TV the music stops because the TV audio takes over? That would be a normal behavior. Or are you playing music on another speaker via Airplay and when you turn on the TV the other speaker also stops?

If true, is your TV and speaker grouped together? They should not be in a group then.

If the problem still exists, please test the latest version 1.0.13.

If the problem also exists with the latest version, please upload your log file here.
